### DISCUSSION TITLE
Why are there two events in rfsrc_fit model ?

### DISCUSSION THREAD

**Discussion by ymy4984** [kogalur/randomForestSRC#391] [Q&A]
I used my own data to fit a model, the code is as follows:
                    rfsrc_fit <- rfsrc(
                      Surv(survmonth,status)~., #公式
                      ntree = 100,         # 树的棵树 
                      nsplit = 5,          # 最小节点数
                      importance = TRUE,   #变量重要性
                      tree.err=TRUE,       # 误差
                      data=train4       # 数据集
                    )
                    print(rfsrc_fit)
However, the results showed there were two events instead of number of death, which was shown as follows:
                    > print(rfsrc_fit)
                                             Sample size: 281
                                        **Number of events: 254, 27**
                                         Number of trees: 100
                               Forest terminal node size: 15
                           Average no. of terminal nodes: 14.58
                    No. of variables tried at each split: 4
                                  Total no. of variables: 11
                           Resampling used to grow trees: swor
                        Resample size used to grow trees: 178
                                                Analysis: RSF
                                                  Family: surv-CR
                                          Splitting rule: logrankCR *random*
                           Number of random split points: 5
                       (OOB) Requested performance error: 0.46319754, 0.41737892
I have verified that I have only two status, namely 0 and 1. And the plot shows two events.

And the second question, when i plotted this model, it errors as follows:
                          > plot.survival(rfsrc
                          +               ) 
                          Error in plot.survival(rfsrc) : 
                            This function only works for objects of class `(rfsrc, grow)' or '(rfsrc, predict)'

**Comment by ishwaran** [comment]
The family being used in your data set problem is `surv-CR` which means a competing risk setting was found -- this is because your `status` variable is not 0/1 which is the default requirement for right-censored survival.  If status takes integer values other than 0/1 then it switches to a [competing risk problem](https://www.randomforestsrc.org/articles/competing.html).

And when you are working with competing risks, in order to plot the results, the function to be used is [plot.competing.risk](https://www.randomforestsrc.org/reference/plot.competing.risk.rfsrc.html)
